routes/web-main.php

```diff
@@ -120,9 +120,7 @@
     'middleware' => 'auth_full_group',
 ], function () {
     Route::get('/')->uses('Registration@show')->name('show');
-    Route::redirect('/invite', config('services.discord.invite_url'))->name('invite');
```

Comment:
@CalumTowers I assumed we don't need these anymore?

Reply:
Don't need the invite URL nope!